langserver/hover.go

```diff
@@ -73,3 +106,42 @@ func packageStatementName(fset *token.FileSet, files []*ast.File, node *ast.Iden
 	}
 	return ""
 }
+
+// maybeAddComments appends the specified comments converted to Markdown godoc
+// form to the specified contents slice, if the comments string is not empty.
+func maybeAddComments(comments string, contents []lsp.MarkedString) []lsp.MarkedString {
+	if comments == "" {
+		return contents
+	}
+	var b bytes.Buffer
+	doc.ToMarkdown(&b, comments, nil)
+	return append(contents, lsp.MarkedString{
+		Language: "markdown",
+		Value:    b.String(),
+	})
+}
+
+// packageDoc finds the documentation for the named package from it's files or
+// a superset of them.
```

Comment:
- Will fix typo.
- superset: Any `[]*ast.File` that is inclusive of the entire packages files, but may include more (e.g. files outside of the package).